apps/web/src/components/Profile/Achievements/StreaksList.tsx

```diff
@@ -109,14 +110,18 @@ const StreaksList: FC<StreaksListProps> = ({ profile }) => {
       </div>
       <div className="divider" />
       <div className="space-y-4 p-6">
-        {data.map((streak: { id: string; event: string; date: string }) => (
-          <div key={streak.id} className="flex items-center space-x-2">
-            <EventIcon event={streak.event} />
-            <div>
-              <EventName event={streak.event} />
-            </div>
-          </div>
-        ))}
+        {data && (
```

Comment:
<img width="888" alt="Screenshot 2023-08-30 at 7 04 36 AM" src="https://github.com/lensterxyz/lenster/assets/69431456/268d71fa-0cf2-425a-8c76-9e76b45312ee">

Spacing is not there